static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -8,8 +8,7 @@ import { getAppliedFilterValue, matchFieldName } from "./util"
 
 export const FILTER_ID_ADJUST = {
   courses:        "program.enrollments.course_title",
-  payment_status: "program.enrollments.payment_status",
-  semester:       "program.enrollments.semester"
+  payment_status: "program.enrollments.payment_status"
```

Comment:
do you think it's worthwhile to add a test for this change, or no? I'm not exactly sure what this `FILTER_ID_ADJUST` thingy is supposed to be doing

Reply:
there is a test for `shows the toggle icon when a filter name matches our filter id map and is a prefix of an ES result key`. We just removed it from here. i dont think so we need a test

Reply:
btw i can add test like
```
  it("shows the toggle icon when a filter name matches our filter id map and is a prefix of an ES result key", () => {
      sandbox.stub(FilterVisibilityToggle.prototype, "getResults").returns({
        aggregations: {
          [`program.enrollments.semester123`]: {
            doc_count: 2
          }
        }
      })
      props.filterName = key
      const wrapper = renderWrappedToggle(props, <div id={key}>Test Text</div>)
      assert.equal(wrapper.find("Icon").props().className, "")
      sandbox.restore()
  })
```